### PR TITLE
Added getter for accessToken in K-Api 

### PR DESCRIPTION
**Pull request by sanatnayar**

### CODE REVIEW

File: pkg/config/config_test.go [hunk 1]
```diff
@@ -190,3 +192,20 @@ func TestGetFromSecrets(t *testing.T) {
 		t.Log(err)
 	}
 }
+
+func TestAccessTokenRetrieval(t *testing.T) {
+	reader := setup(t)
```
Comment (ffoysal) [inline pkg/config/config_test.go]: should skip here

File: pkg/config/config_test.go [hunk 2]
```diff
@@ -190,3 +192,20 @@ func TestGetFromSecrets(t *testing.T) {
...
+	cfg, _ := ReadCRSpecFromFile(reader)
+	if cfg.Spec.Git.SecretName != "" {
+		// skipped because need kubectl (will not perform ci checks)
+		// if need to test, comment line bellow and add secretName to example spec
+		t.Skip()
+	} else {
+		cfg.Spec.AddToSecrets("qliksense2", "mongo", "tadadaa", "")
+		if _, err := cfg.Spec.Git.GetAccessToken(); err != nil {
```
Comment (ffoysal) [inline pkg/config/config_test.go]: in your test create the secrets kubectl first

**Comment by ffoysal** [comment]
from inside your tests:
- create secrets name `test-access-token`
- add that secret name in CR config
- Try to read the accessToken using your function
- if it finds the secret then PASS

File: pkg/config/config_test.go (hunk 2 above)
Comment (sanatnayar) [reply pkg/config/config_test.go]: created secret

File: pkg/config/config_test.go (hunk 1 above)
Comment (sanatnayar) [reply pkg/config/config_test.go]: added skip

File: pkg/config/config.go [hunk 3]
```diff
@@ -149,6 +149,7 @@ func (nv NameValue) GetSecretValue() string {
 }
 
 func readFromKubernetesSecret(secName, keyName string) (string, error) {
+	log.Println("hi")
```
Comment (ffoysal) [inline pkg/config/config.go]: is it ??

Comment (sanatnayar) [reply pkg/config/config.go]: sorry, forgot to remove that, was using it to see if the function was being called

**Pull request merged**